test/image-comparison.js

```diff
@@ -23,53 +23,59 @@ const expect = require('chai').expect;
 const debug = require('debug')('tester');
 const colors = require('colors');
 
+const PNG = require('pngjs').PNG;
+
 var rewire = require('rewire'); // for testing unexported functions, see https://stackoverflow.com/questions/14874208/how-to-access-and-test-an-internal-non-exports-function-in-a-node-js-module
 var checkerCore = rewire('../lib/checker.js');
 
-runBlinkDiff = checkerCore.__get__('runBlinkDiff');
-sliceImagesOutOfHTMLStringsAndCreateBuffers = checkerCore.__get__('sliceImagesOutOfHTMLStringsAndCreateBuffers');
+compareImages = checkerCore.__get__('compareImages');
+extractImagesOutOfHTMLStringsAndCreateBuffers = checkerCore.__get__('extractImagesOutOfHTMLStringsAndCreateBuffers');
+prepareImagesForComparison = checkerCore.__get__('prepareImagesForComparison');
+
 
 describe('Testing image comparison', function () {
 
-	describe('Comparison via blink-diff', function () {
+	describe('Testing image comparison', function () {
 		var paperA = 'test/TestPapers_2/paper_9_img_A.html';
-		var paperB = 'test/TestPapers_2/paper_9_img_C.html';
+		var paperB = 'test/TestPapers_2/paper_9_img_D.html';
 
 		var images;
 
 		try {
 			fs.mkdirSync(path.join(os.tmpdir(), 'erc-checker'));
-		}catch (e) {}
-
+		} catch (e) { debug(e); }
 
 		before(function (done) {
-
+			//this.timeout(60000);
```

Comment:
Please remove commented lines like this one.

Reply:
Is removed